docs/sources/flow/reference/components/prometheus.remote_write.md

```diff
@@ -332,3 +332,7 @@ prometheus.scrape "demo" {
 ## Compression
 
 `prometheus.remote_write` uses [snappy](https://en.wikipedia.org/wiki/Snappy_(compression)) for compression.
+
+## Meta labels
+
+Any labels that start with `__` will be removed before sending to the endpoint. 
```

Comment:
Does this belong as a dedicated H2, or should it go closer to the documentation for where data is sent (the `endpoint` block)?

I'm concerned that adding too many h2s breaks the consistency patterns of how we document reference pages for components. 

cc @clayton-cornell

Reply:
I am fine with that too, I think there is an unknown in where to put these little implementation details.

Reply:
Could this be formatted as a note?

Reply:
Is there a place for something like rolling up the compression/meta into this? Though it feels very grab bag. 

```
## Implementation details

* `prometheus.remote_write` uses [snappy](https://en.wikipedia.org/wiki/Snappy_(compression)) for compression.
* Any labels that start with `__` will be removed before sending to the endpoint.
```

Its almost like a FAQ or troubleshooting guidelike. The entry point might be why are my labels not being written to prometheus or what compression does remote_write use

Reply:
I've been thinking about this, and don't yet have a suggestion.

* What would a user be doing with Agent where they'd need to know this info? Initial config? Fine tuning? Trying to fix a problem with data flow/collection?
* Is this simply nice to know info? or more critical to a successful Agent setup?

Reply:
Its a common gotcha that people remark on. Usually like where are my labels? The compression one is more from a customer asking details to make sure we arent writing 5x the data we should.

Reply:
@mattdurham This PR is getting old. I suggest we merge it as-is even though it's not 100% what we'd like. Get the info at least published and we can revisit this as it evolves.